common/lib/driver-definitions/src/storage.ts

```diff
@@ -312,8 +320,16 @@ export interface IDocumentServiceFactory {
 
     /**
      * Creates a new document with the provided options. Returns the document service.
+     *
+     * Note: it is the consumer's responsibility to cleanup
```

Comment:
Can you please explain what do you mean by "it is consumer's responsibility to cleanup"?